daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/debugger/InteractiveDebugger.scala

```diff
@@ -1837,7 +1837,7 @@ class InteractiveDebugger(
         val longDesc = desc
 
         def getSomeValue(state: StateForDebugger): Option[Long] = {
-          if (state.arrayPos != -1) Some(state.arrayPos) else None
+          if (state.arrayIterationPos != -1) Some(state.arrayIterationPos) else None
```

Comment:
Should this return `occursIndexPos`? The debugger command is called `occursIndex` so that might be what users expect?

Reply:
This was an auto-refactor here. I went ahead and updated it to use occursPos.